### PR TITLE
Add new package: byte-unixbench

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/byte-unixbench/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class ByteUnixbench(Package):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/byte-unixbench/package.py]:
```suggestion
class ByteUnixbench(MakefilePackage):
```

File: var/spack/repos/builtin/packages/byte-unixbench/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    homepage = "https://code.google.com/archive/p/byte-unixbench"
+    url      = "https://github.com/kdlucas/byte-unixbench/archive/v5.1.3.tar.gz"
+
+    version('5.1.3', sha256='3a6bb00f270a5329682dff20fd2c1ab5332ef046eb54a96a0d7bd371005d31a3')
+
+    build_directory = 'UnixBench'
+
+    def install(self, spec, prefix):
+        with working_dir(self.build_directory):
+            make()
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/byte-unixbench/package.py]: You can delete this line with the above change

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/byte-unixbench/package.py (hunk 2 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/byte-unixbench/package.py]: Fixed it in https://github.com/spack/spack/pull/18257/commits/317f9ed0c09ddd8e1eb01d8df90f65b618766e17

**Review by adamjstewart — APPROVED** [review]

File: var/spack/repos/builtin/packages/byte-unixbench/package.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    homepage = "https://code.google.com/archive/p/byte-unixbench"
+    url      = "https://github.com/kdlucas/byte-unixbench/archive/v5.1.3.tar.gz"
+
+    version('5.1.3', sha256='3a6bb00f270a5329682dff20fd2c1ab5332ef046eb54a96a0d7bd371005d31a3')
+
+    build_directory = 'UnixBench'
+
+    def install(self, spec, prefix):
+        install_tree('.', prefix)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/byte-unixbench/package.py]: Shouldn't this be:
```suggestion
        with working_dir(self.build_directory):
            install_tree('.', prefix)
```

**Pull request merged**